services/user/Sites/plugin/wit/world.wit

```diff
@@ -16,13 +16,19 @@ interface sites {
     /// 
     /// This file will be accessible at the subdomain of the caller, 
     /// at the specified path.
+    /// 
+    /// Maximum quality Brotli file compression is automatically applied to 
+    /// the file if compression on its content type is supported.
```

Comment:
Why is the default different from the CLI?

Reply:
I think I was thinking CLI is more likely to be used in scripts for uploading large amounts of content, in which case level 11 can be quite slow.

Maybe they should just be the same. Also, not sure why I didn't parameterize it here. I'll address both of those

Reply:
Maybe a front-end GUI should default to max compression, because it's more likely to be used by people who will just accept the default, and they might as well get max compression.